warehouse/core/src/main/java/datawave/ingest/util/cache/Loader.java

```diff
@@ -165,6 +165,10 @@ public void run() {
         
     }
     
+    public boolean equals(Loader loader) {
+        return (this.hashCode() == loader.hashCode());
+    }
+    
```

Comment:
This is not the equals method you are looking for.  Also if we are to actually match the intent of the hashCode method below, then this needs to actually be the same object.
```suggestion
    @Override
    public boolean equals(Object loader) {
        return this == loader;
    }
    
```